src/ql/ir/compat/detail/cqasm_reader.h

```diff
@@ -19,12 +19,12 @@ namespace detail {
 
 // Shorthands for namespaces.
 namespace lqt { using namespace ::cqasm::tree; }
-namespace lqs { using namespace ::cqasm::semantic; }
-namespace lqv { using namespace ::cqasm::values; }
-namespace lqtyp { using namespace ::cqasm::types; }
-namespace lqi { using namespace ::cqasm::instruction; }
-namespace lqa { using namespace ::cqasm::analyzer; }
-namespace lqp { using namespace ::cqasm::parser; }
+namespace lqs { using namespace ::cqasm ::v1x::semantic; }
```

Comment:
spurious space?

Reply:
Wow! I wonder how that code was working! 😮